aten/src/ATen/native/native_functions.yaml

```diff
@@ -903,16 +903,33 @@
 
 - func: clamp(Tensor self, Scalar? min=None, Scalar? max=None) -> Tensor
   variants: function, method
+  cpp_no_default_args: ['min']
   dispatch:
     CPU, CUDA: clamp
     QuantizedCPU: clamp_quantized_cpu
 
+- func: clamp.Tensor(Tensor self, Tensor? min=None, Tensor? max=None) -> Tensor
+  variants: function, method
+  dispatch:
+    CPU, CUDA: clamp
```

Comment:
NB: CPU, CUDA designation here is safe, but actually it is too restrictive as the clamp implementation is

```
Tensor clamp(const Tensor& self, const c10::optional<Tensor>& min, const c10::optional<Tensor>& max) {
  Tensor result = at::empty({0}, self.options());
  return at::clamp_outf(self, min, max, result);
}
```

which is perfectly device generic. However, I don't think you need to fix this; it will get automatically fixed if we port clamp to structured (which, honestly, we probably should have done before this PR... too late for that though lol)